cmd/argo/commands/list.go

```diff
@@ -53,6 +54,11 @@ func NewListCommand() *cobra.Command {
 			}
 			workflows, err := listWorkflows(ctx, serviceClient, listArgs)
 			errors.CheckError(err)
+			if len(workflows) == 0 {
+				fmt.Printf("No resources found\n")
```

Comment:
"no workflows found"?

Reply:
move into "PrintWorkflows" and add test?

Reply:
ok